QuizApp/QuizSession/Views/QuestionView.swift

```diff
@@ -44,8 +44,13 @@ class QuestionView: UIView {
                 attributes: [.font: UIFont.systemFont(ofSize: 20, weight: .semibold),
                              .foregroundColor: UIColor.white]
             )
-            answerButton.titleLabel?.numberOfLines = 0
-            answerButton.titleLabel?.lineBreakMode = .byWordWrapping
+
+            guard let answerTitleLabel = answerButton.titleLabel else {
+                return
+            }
```

Comment:
fits in one line
you're still force unwrapping below: `answerButton.titleLabel!.snp.height`